etna/ciim/models.py

```diff
@@ -152,7 +152,7 @@ def __getitem__(self, key):
             return self.model(**transformed_result)
 
         self._fetch(start=key.start, size=count)
-        translated_results = [self.model.transform(r) for r in self._results[key]]
+        translated_results = [self.model.transform(r) for r in self._results[key] if r is not None]
```

Comment:
Thanks for this @mozaki1! Looks good. 

Thanks for updating the record fixture and the transform function.

Is this amend to filter out `None` and/or the amend to the chooser something you'd be able to write a test for?